.github/workflows/tex-tests.yml

```diff
@@ -27,16 +27,16 @@ jobs:
         # For example -- os: [ubuntu-latest, macos-latest, windows-latest]
         os: [ubuntu-latest, macos-latest, windows-latest]
 
-        # Test python 3.8, 3.9, and 3.10 by default.
-        python-ver: [8, 9, 10]
+        # Test python version 3.x
+        python-ver: [9, 10, 11]
```

Comment:
Should we add 12? It was released in october.

Reply:
I had some issues with tests with 12 in other repositories.  So maybe delay for now.